alpha/lib/model/entry.php

```diff
@@ -1979,6 +2021,20 @@ public function isEntitledKuserEdit($kuserId)
 
 		return $this->isOwnerActionsAllowed($kuserId);
 	}
+	
+	public function isEntitledKuserView($kuserId)
+	{
+		$entitledKuserArray = array_keys($this->getEntitledPusersViewArray());
```

Comment:
Consolidate the common code of this function and isEntitledKuserEdit since most of it is the same.